test/index.js

```diff
@@ -0,0 +1,51 @@
+import {
+  lspPositionToUnistPoint,
+  lspRangeToUnistPosition,
+  unistPointToLspPosition,
+  unistPositionToLspRange
+} from 'unist-lsp'
+import test from 'tape'
+
+test('unistPointToLspPosition', async (t) => {
```

Comment:
Probably good to tests missing props? null/undefined/invalid props?

Reply:
IMO it’s fine to rely on type information for this.

Reply:
I won’t veto this but generally I believe that it is important to care for users who don’t use TypeScript.
I also believe that, for an open source library, it is important to write “defensively”. As in, assume people pass weird things in, and to an extent, handle that.
People do weird things in plugins. It’s good to not crash on weird things.

Reply:
I think it depends a bit. I.e. when writing a remark plugin I tend to care about this more then when writing such low-level functions with a specific use cases like these.